.github/labeler.yml

```diff
@@ -21,7 +21,6 @@ common/:
   - resources/**
 common/core/: common/core/**
 common/core/desktop/: common/core/desktop/**
-common/core/web/: common/core/web/**
```

Comment:
No `common/web/`?  Just eliminating the label entirely?

Reply:
Hmm, not entirely sure. I have another round of tweaks to labeling in https://github.com/keymanapp/keyman/pull/6629/files#diff-a22c263686553013feaeb0677d26eeb0b8778a756c4311c1fce13384258026aa but just realised that has a typo in it...